packages/one-app-bundler/README.md

```diff
@@ -164,6 +164,37 @@ under `bundler.purgecss.paths`. The example below illustrates how we would add
 }
 ```
 
+Additional `purgecss` options. Please refer to the [`purgecss Options Documentation`](https://github.com/FullHuman/purgecss)
+before enabling any of the following:
+
+<!-- #TODO: How to pass regex instead of string and how to pass a library to the extractor instead of string like "purgeJs" -->
+
+```json
+{
+  "one-amex": {
+      "bundler": {
+        "purgecss": {
+        "paths": ["node_modules/some-lib/src/**/*.{js,jsx}"],
```

Comment:
can you check the indentation

Reply:
solved here https://github.com/americanexpress/one-app-cli/pull/61/commits/3eea2fc005ac7c4257602f054509398258cb9610